src/electron/worker.js

```diff
@@ -152,76 +127,64 @@ class NspvBitGoSingleton {
    * }
    */
   async listUnspent(address) {
-    try {
-      if (!this.connected) {
-        await this.connect();
-      }
-      const response = await ccutils.getNormalUtxos(this.peers, address, 0, 0);
-      const ccUtxos = await ccutils.getCCUtxos(this.peers, address, 0, 0);
-      return {
-        height: response.height,
-        skipcount: response.skipcount,
-        filter: response.filter,
-        lastpeer: response.lastpeer,
-        cc: ccUtxos,
-        balance: response.total / SATOSHIS,
-        numutxos: response.utxos.length,
-        address: this.address,
-      };
-    } catch (e) {
-      console.error(e);
-      throw new Error(e);
+    if (!this.connected) {
+      throw new Error('Not connected');
     }
+    const response = await ccutils.getNormalUtxos(this.peers, address, 0, 0);
```

Comment:
Should await these in parallel with `Promise.all`.

Reply:
actually need to check if we can use that getCCUtxos function at all